.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,17 @@
+Link to the milestone on Github e.g. https://github.com/nimblehq/infrastructure-templates/milestone/41?closed=1
+or
+Link to the project management tool for the release
+
+## Features
+
+Provide the ID and title of the issue in the section for each type (feature, chore and bug). The link is optional.
+
+- [ch1234] As a user, I can log in
+or
+- [[ch1234](https://github.com/nimblehq/infrastructure-templates/issues/1234)] As a user, I can log in
+
+## Chores
+- Same structure as in  ## Feature
```

Comment:
Is there a redundant space here?
```suggestion
- Same structure as in ## Features
```

Reply:
Updated in https://github.com/nimblehq/infrastructure-templates/pull/21/commits/e1e2e2961b0ad3409eb0b7e969fe9a28b27649fe